app/routes/organen/orgaan/edit.js

```diff
@@ -4,22 +4,17 @@ import { getFormFrom } from 'frontend-lmb/utils/get-form';
 import { BESTUURSORGAAN_FORM_ID } from 'frontend-lmb/utils/well-known-ids';
 import RSVP from 'rsvp';
 
-export default class OrganenBeheerEditRoute extends Route {
+export default class OrganenEditRoute extends Route {
   @service store;
-  async model(params) {
-    const bestuursorgaan = this.store.findRecord(
-      'bestuursorgaan',
-      params.orgaan_id,
-      {
-        include: 'classificatie,heeft-tijdsspecialisaties',
-      }
-    );
+
+  async model() {
+    const parentModel = this.modelFor('organen.orgaan');
     const formDefinition = getFormFrom(this.store, BESTUURSORGAAN_FORM_ID);
 
     return RSVP.hash({
```

Comment:
Question: What does the RSV.hash do and why did you remove it in the beheer.index route ?

Reply:
RSVP.hash is used whenever you return multiple objects in your model which are promises. The RSVP.hash is then used for handling these promises, I was cleaning up a bit trying to fix then jittering when rerendering, but it is probably safer to leave this.